src/main.ts

```diff
@@ -5,8 +5,9 @@ import xrayExpress from 'aws-xray-sdk-express';
 import express from 'express';
 import https from 'https';
 import { graphqlUploadExpress } from 'graphql-upload';
-import { server as adminServer } from './admin/server';
+import { startAdminServer } from './admin/server';
 import { server as publicServer } from './public/server';
+import { getContext } from './admin/context';
```

Comment:
could we be consistent on naming/importing things from `admin`? i think either of these would work:

```
import { startServer as startAdminServer } from './admin/server';
import { getContext as getAdminContext } from './admin/context';
```

```
import { startAdminServer } from './admin/server';
import { getAdminContext } from './admin/context';
```

Reply:
Fixed!